addons/website/static/src/builder/plugins/form/form_option_plugin.js

```diff
@@ -155,9 +155,8 @@ export class FormOptionPlugin extends Plugin {
             SetVisibilityDependencyAction,
             SetFormCustomFieldValueListAction,
         },
-        system_classes: ["o_builder_form_show_message"],
```

Comment:
Don't you think it's strange to have a step in the history for this kind of technique? We should probably just trigger the dom_update event even though we don't have a step in the history, don't you think?